docs-md/tutorials/clickstream-docker.md

```diff
@@ -323,11 +323,11 @@ SELECT * FROM PAGES_PER_MIN EMIT CHANGES LIMIT 5;
 Your output should resemble:
 
 ```
-1536662784977 | 21 : Window{start=1536662725000 end=-} | 21 | 2
-1536662789353 | 21 : Window{start=1536662730000 end=-} | 21 | 7
-1536662793715 | 21 : Window{start=1536662735000 end=-} | 21 | 20
-1536662799627 | 21 : Window{start=1536662740000 end=-} | 21 | 35
-1536662804534 | 21 : Window{start=1536662745000 end=-} | 21 | 40
+1536662784977 | 1536662725000 | 1536662730000 | 21 | 21 | 2
+1536662789353 | 1536662730000 | 1536662735000 | 21 | 21 | 7
+1536662793715 | 1536662735000 | 1536662740000 | 21 | 21 | 20
+1536662799627 | 1536662740000 | 1536662745000 | 21 | 21 | 35
+1536662804534 | 1536662745000 | 1536662750000 | 21 | 21 | 40
```

Comment:
Updated to reflect changes in `ROWKEY` and addition of `WINDOWSTART` and `WINDOWEND`.